ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml

```diff
@@ -58,14 +58,21 @@ images:
 promotion:
   name: "4.9"
   namespace: ocp
+releases:
+  initial:
+    release:
+      channel: candidate
+      version: "4.8"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.9"
 resources:
   '*':
     requests:
       cpu: 100m
       memory: 200Mi
-tag_specification:
```

Comment:
Don't remove `tag_specification` in `openshift-ovn-kubernetes-master.yaml`. 

Instead copy the contents to `openshift-ovn-kubernetes-master__4.9-upgrade-from-4.8.yaml`.
This will create `4.9-upgrade-from-4.8` variant of the job there, where custom releases (and no `tag_specification`) can be set. See [`openshift-ovn-kubernetes-master__4.8-upgrade-from-stable-4.7.yaml`](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master__4.8-upgrade-from-stable-4.7.yaml)